llvm/lib/Target/AMDGPU/GCNRegPressure.cpp

```diff
@@ -88,8 +88,8 @@ void GCNRegPressure::inc(unsigned Reg,
   }
 }
 
-bool GCNRegPressure::less(const GCNSubtarget &ST,
-                          const GCNRegPressure& O,
+bool GCNRegPressure::less(const GCNSubtarget &ST, const MachineFunction &MF,
```

Comment:
can you leave only MF parameter and obtain ST from it, like

`const GCNSubtarget &ST = MF.getSubtarget<GCNSubtarget>();`